content/tutorials/marketplace.md

```diff
@@ -3,43 +3,19 @@ title: Set Up a Marketplace
 description:
 ---
 
-In Ocean, marketplaces and publishers are different roles. A common setup is for one organization to do both. We focus on that here.
+## About marketplace
 
-## The Steps
+Ocean Protocol's [marketplace](https://market.oceanprotocol.com/) provider a web interface for accessing assets published on the chain. By default, the assets are pulled from Aquarius hosted by Ocean Protocol. To extend the existing features of the marketplace, developers can fork the marketplace.
```

Comment:
"provider a web interface" -> "provides a web interface"
"By default, the assets are pulled from Aquarius hosted by Ocean Protocol" this just doesn't sound right. It sounds like the OceanProtocol hosts the assets which is incorrect

Reply:
how about ` By default, the metadata of the assets are pulled from Aquarius hosted by Ocean Protocol.`?